CONTRIBUTING.md

```diff
@@ -1,10 +1,14 @@
 # How to contribute
 
-One of the easiest ways to contribute is to participate in discussions on GitHub issues. You can also contribute by submitting pull requests with code changes.
+One of the easiest ways to contribute is to participate in discussions on GitHub issues and provide feedback. You can also contribute by submitting pull requests with code changes.
 
 ## Feedback, bug reports, and feature requests
 
-Please start a discussion in the [dotnet/razor-tooling issue tracker](https://github.com/dotnet/razor-tooling/issues).
+You can report issues and make suggestions in the [dotnet/razor issue tracker](https://github.com/dotnet/razor/issues).
+
+If you're hitting issues with the Razor tooling in Visual Studio, you can let us know using the Visual Studio "Send Feedback" feature (Help > Send Feedback), which will automatically include helpful logs and diagnostic information for troubleshooting purposes. Most Visual Studio feedback issues will then get migrated to the dotnet/razor GitHub repo for tracking purposes.
+
+If someone has already reported your issue or suggestion, be sure to give it a :+1: on the original post so that we know it's important to you! You can also comment on issues  with clarifying details on your scenarios and the impact the issue has.
```

Comment:
```suggestion
If someone has already reported your issue or suggestion, be sure to give it a :+1: on the original post so that we know it's important to you! You can also comment on issues with clarifying details on your scenarios and the impact the issue has.
```